src/routes/assistants/+page.server.ts

```diff
@@ -34,5 +43,7 @@ export const load = async ({ url }) => {
 		assistants: JSON.parse(JSON.stringify(assistants)) as Array<Assistant>,
 		numTotalItems,
 		numItemsPerPage: NUM_PER_PAGE,
+		createdByUser: createdByName,
+		createdByMe: createdByName === (locals.user?.username ?? locals.user?.name),
```

Comment:
No need for this I think, just compare the requested username with the current username in the page directly to see it's the same?

Reply:
handled in https://github.com/huggingface/chat-ui/pull/773/commits/6913adc8483fcfc4a7ae34c2627f03c38aaab3df